packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -53,6 +55,10 @@ export default class OverTheBoxDetailsCtrl {
     this.allDevices = [];
     this.device = null;
 
+    // Retrieve user language to get guides link by language
+    this.language = this.coreConfig.getUserLanguage();
+    this.guidesLink = this.OVERTHEBOX_DETAILS.guidesUrl[this.language];
```

Comment:
Usually we map guides with subsidiary and not language, any reason why the behaviour is changed here ?

Reply:
I'm not sure that there is a subsidiary for OTB.